block/manager.go

```diff
@@ -125,7 +126,8 @@ func NewManager(
 		blockCache: &Cache{
 			cache: make(map[uint64]types.CachedBlock),
 		},
-		pruningC: make(chan int64, 10), // use of buffered channel to avoid blocking applyBlock thread. In case channel is full, pruning will be skipped, but the retain height can be pruned in the next iteration.
+		FraudHandler: nil,                  // TODO: create a default handler
```

Comment:
it's required to fix, otherwise `applyBlockWithFraudHandling` will panic on fraud